src/components/Board/Board.component.js

```diff
@@ -104,7 +107,71 @@ export class Board extends Component {
 
     this.state = {
       openTitleDialog: false,
-      titleDialogValue: props.board && props.board.name ? props.board.name : ''
+      titleDialogValue: props.board && props.board.name ? props.board.name : '',
+      runLiveHelp: false,
+      runLiveUnlock: false,
+      hideCloseButton: true,
+      steps: [
```

Comment:
We want to keep the state as small as we can (performance reasons), and since the joyride steps are static and do not change it is better to keep the object a property on the class and reference it later in the code as `this.steps`.

Reply:
You are right! I made both steps class properties, and modified their names to unlockedHelpSteps & lockedHelpSteps as we talked about!